resources/mainnet/seeds.txt

```diff
@@ -1 +0,0 @@
-3d67d0646cddcc203b41434aceea64ade22ba6fc@k8s-mainnet-axelarco-79b464ee93-f03cb16c57cf7cb2.elb.us-east-2.amazonaws.com:26656,9c9b8eb8cad8648f31d3c9da3d14e825f11a7b52@public-seed.mainnet.axelar.lunanova.tech:26656
```

Comment:
Mainnet upgrade will only occur next week. So, this seeds.txt is still needed in the meantime.

Reply:
The seeds.txt files in axelarate-community are not used anywhere.  Instead the join scripts download them from S3.  So we need to keep them on S3 (for now at least) but not here.